packages/ui/src/ui/store/reducers/navigation/modals/tableMergeSortModalSlice.ts

```diff
@@ -0,0 +1,38 @@
+import {PayloadAction, createSlice} from '@reduxjs/toolkit';
```

Comment:
it is better to use `git mv` instead of removing `table-merge-sort-modal.ts`

Reply:
everything is prescribed here from 0

Reply:
But it is still the same reducer just rewritten with createSlice